task-1-STREAMS/Tasks.md

```diff
@@ -0,0 +1,29 @@
+# LeverX Stream Task
+
+## Important information:
+1) You can find some useful materials in [Materials.md](materials/Materials.md);
+2) Pay attention to the file [Self-check.md](Self-check.md);
+3) You are able to use Lombok.
+
+## Task description:
```

Comment:
Let's rename this title to "Requirements"

Reply:
@Shotim @IlyaBritkov but it is not requirements...

Reply:
Let's just call it like this. Additionally, there could be exactly requirements in the future